ci/terraform/oidc/ipv-callback.tf

```diff
@@ -35,23 +35,23 @@ module "ipv-callback" {
     ACCOUNT_INTERVENTION_SERVICE_AUDIT_ENABLED = var.account_intervention_service_audit_enabled
     ACCOUNT_INTERVENTION_SERVICE_ENABLED       = var.account_intervention_service_enabled
     ACCOUNT_INTERVENTION_SERVICE_URI           = var.account_intervention_service_uri
-    DYNAMO_ENDPOINT                 = var.use_localstack ? var.lambda_dynamo_endpoint : null
-    ENVIRONMENT                     = var.environment
-    IDENTITY_ENABLED                = var.ipv_api_enabled
-    INTERNAl_SECTOR_URI             = var.internal_sector_uri
-    IPV_AUDIENCE                    = var.ipv_audience
-    IPV_AUTHORISATION_CALLBACK_URI  = var.ipv_authorisation_callback_uri
-    IPV_AUTHORISATION_CLIENT_ID     = var.ipv_authorisation_client_id
-    IPV_AUTHORISATION_URI           = var.ipv_authorisation_uri
-    IPV_BACKEND_URI                 = var.ipv_backend_uri
-    IPV_NO_SESSION_RESPONSE_ENABLED = var.ipv_no_session_response_enabled
-    IPV_TOKEN_SIGNING_KEY_ALIAS     = local.ipv_token_auth_key_alias_name
-    LOCALSTACK_ENDPOINT             = var.use_localstack ? var.localstack_endpoint : null
-    LOGIN_URI                       = "https://${local.frontend_fqdn}/"
-    OIDC_API_BASE_URL               = local.api_base_url
-    REDIS_KEY                       = local.redis_key
-    SPOT_QUEUE_URL                  = aws_sqs_queue.spot_request_queue.id
-    TXMA_AUDIT_QUEUE_URL            = module.oidc_txma_audit.queue_url
+    DYNAMO_ENDPOINT                            = var.use_localstack ? var.lambda_dynamo_endpoint : null
```

Comment:
Would be nicer to squash the formatting changes into the previous commit, can show you how if you like